server/routes/eligibility-checker/how-certain.route.js

```diff
@@ -30,6 +30,12 @@ const handlers = {
       RedisKeys.USED_CHECKER,
       payload.howCertain !== completelyCertain
     )
+    await request.ga.event({
```

Comment:
Wondering if we need to await this or can we just fire and forget?

Reply:
II think it may hang while it's trying to communicate with analytics if removed. From what I've read about the plugin it may batch up several requests before sending to analytics, not sure if it does this though. I put in the `await` in there as that's how they have done it in the examples for the plugin. I'll remove it and see what happens.

I'll get an analytics account set up on my own Google account to try it out. I didn't really have the time to do this before I went on leave but it would be useful to see what it does/if it works before we push the change to dev/test.

Cheers

Reply:
Just noticed you've approved it, should have scrolled down further :) I'll give what I've suggested above a go anyway and let you know what happens before I merge it in.